dashboard/dashboard-front/src/components/jobs/JobsListView.vue

```diff
@@ -131,6 +133,34 @@ watch(jobsTree, () => {
     })
 })
 
+watch(envInfo, () => {
+    setupEventStreamClient()
+})
+
+function setupEventStreamClient() {
+    if (!envInfo.lifecycle_url)
+        return
+
+    const url = new URL(envInfo.lifecycle_url)
+    url.pathname = ''
+    const trimmedLifecycleUrl = url.toString()
+    const socket: Socket = io(trimmedLifecycleUrl, {
+        path: '/lifecycle/socketio/events',
+    })
+
+    socket.on("connect", () => {
+        console.log(`connected to live events stream: ${socket.id}`)
+        socket.on("broadcast_event", (data) => {
+            console.log('live event received:', data)
```

Comment:
Also it would be nice UX to show on page that the list has been updated... ie. flash small green circle somewhere in corner. Or just static text there "Last state update: 7 seconds ago". 

Btw, do you have it running somewhere so I can test?

Reply:
Yes, I've added a notification. 

You can test it now. It is running on infamous ireneusz1 cluster.

Reply:
Plase reset password for user hubert.rutkowski@deepsense.ai ^^

Reply:
Ok, was able to test it. Indeed, works! Also on FF. Very cool :) And yeah, the toast messages are a bit irritating.

My final suggestion is to remove some logging, because it spams the console, while not providing useful data:
![image](https://github.com/TheRacetrack/racetrack/assets/5213082/008103a5-9c83-44ff-b1e5-0ff1eb4def21)